Documentation/event-reference.md

```diff
@@ -23,31 +23,32 @@ The following events are handled by the Edge extension client-side.
 
```

Comment:
### Edge request content

 This event is a request to process and deliver an Experience event to Edge Network. This event is captured by the Edge Network extension's event listener in the Event Hub for processing and sent to Edge Network.

 If the required `xdm` key is not present in the event data payload, the event is not sent to Edge Network. To learn more about Experience Data Model (XDM), please read the [XDM system overview](https://experienceleague.adobe.com/docs/experience-platform/xdm/home.html)​.

 #### Event dispatched by
 * [`Edge.sendEvent(ExperienceEvent experienceEvent, EdgeCallback callback)`](api-reference.md#sendevent)

 #### Event details<!-- omit in toc -->

 | Event type | Event source |
 | ---------- | ------------ |
 | com.adobe.eventType.edge | com.adobe.eventSource.requestContent |

 #### Event data payload definition<!-- omit in toc -->

 | Key | Value type | Required | Description |
 | --- | ---------- | -------- | ----------- |
 | xdm | <code>Map<String,&nbsp;Object></code> | Yes | XDM formatted data; use an `XDMSchema` implementation for better XDM data ingestion and data format control. |
 | data | <code>Map<String,&nbsp;Object></code> | No | Optional free-form data associated with this event. |
 | datasetId | `String` | No | Optional custom dataset ID. If not set, the event uses the default Experience dataset ID set in the datastream configuration. |

 > **Note**  
 > Events of this type and source are only processed if the data collection consent status stored in the `collect` property is **not** `n` (no); that is, either `y` (yes) or `p` (pending).